test/commands/passphrase/encrypt.test.js

```diff
@@ -0,0 +1,196 @@
+	};
+	const encryptedPassphraseObject = {
+		salt: 'salt',
+		cipherText: 'cipherText',
+		iv: 'iv',
+		tag: 'tag',
+		version: 1,
+	};
+	const defaultInputs = {
+		passphrase:
+			'enemy pill squeeze gold spoil aisle awake thumb congress false box wagon',
+		password: 'LbYpLpV9Wpec6ux8',
+	};
+
+	const printMethodStub = sandbox.stub();
+	const setupTest = () =>
+		test
+			.stub(print, 'default', sandbox.stub().returns(printMethodStub))
+			.stub(config, 'getConfig', sandbox.stub().returns({}))
+			.stub(cryptography, 'getKey', sandbox.stub().returns(defaultKeys))
+			.stub(
+				cryptography,
+				'encryptPassphraseWithPassword',
+				sandbox.stub().returns(encryptedPassphraseObject),
+			)
+			.stub(
+				cryptography,
+				'stringifyEncryptedPassphrase',
+				sandbox.stub().returns(encryptedPassphraseString),
+			)
+			.stub(
+				getInputsFromSources,
+				'default',
+				sandbox.stub().resolves(defaultInputs),
+			)
+			.stdout();
+
+	describe('passphrase:encrypt', () => {
+		setupTest()
+			.command(['passphrase:encrypt'])
+			.it('should encrypt passphrase', () => {
+				expect(
+					cryptography.encryptPassphraseWithPassword,
+				).to.be.calledWithExactly(
+					defaultInputs.passphrase,
+					defaultInputs.password,
+				);
+				expect(
+					cryptography.stringifyEncryptedPassphrase,
+				).to.be.calledWithExactly(encryptedPassphraseObject);
+				expect(getInputsFromSources.default).to.be.calledWithExactly({
+					passphrase: {
+						source: undefined,
+						repeatPrompt: true,
+					},
+					password: {
+						source: undefined,
+						repeatPrompt: true,
+					},
+				});
+				return expect(printMethodStub).to.be.calledWithExactly({
+					encryptedPassphrase: encryptedPassphraseString,
+				});
+			});
+	});
+
+	describe('passphrase:encrypt --outputPublicKey', () => {
+		setupTest()
+			.command(['passphrase:encrypt', '--outputPublicKey'])
+			.it('should encrypt passphrase and output public key', () => {
+				expect(
+					cryptography.encryptPassphraseWithPassword,
+				).to.be.calledWithExactly(
+					defaultInputs.passphrase,
+					defaultInputs.password,
+				);
+				expect(
+					cryptography.stringifyEncryptedPassphrase,
+				).to.be.calledWithExactly(encryptedPassphraseObject);
+				expect(getInputsFromSources.default).to.be.calledWithExactly({
+					passphrase: {
+						source: undefined,
+						repeatPrompt: true,
+					},
+					password: {
+						source: undefined,
+						repeatPrompt: true,
+					},
+				});
+				return expect(printMethodStub).to.be.calledWithExactly({
+					encryptedPassphrase: encryptedPassphraseString,
+					...defaultKeys,
+				});
+			});
+	});
+
+	describe('passphrase:encrypt --passphrase=pass:enemy pill squeeze gold spoil aisle awake thumb congress false box wagon', () => {
+		setupTest()
+			.command([
+				'passphrase:encrypt',
+				'--passphrase=pass:enemy pill squeeze gold spoil aisle awake thumb congress false box wagon',
```

Comment:
This isn't exactly how a user would provide this. It would be `--passphrase="pass:enemy pill squeeze gold spoil aisle awake thumb congress false box wagon"`